packages/back-office-subscribers/lib/prisma.js

```diff
@@ -0,0 +1,15 @@
+const { PrismaClient } = require('@prisma/client');
+
+let prismaClientInstance;
+
+const createPrismaClient = () => {
+	if (!prismaClientInstance) {
+		prismaClientInstance = new PrismaClient({
+			log: [{ emit: 'event', level: 'query' }, 'info', 'warn', 'error']
```

Comment:
can be simplified to:
```suggestion
			log: ['query', 'info', 'warn', 'error']
```
as we are not using the `emit: 'event'` functionality (using a custom event handler)